app/controllers/customers_controller.rb

```diff
@@ -242,7 +262,16 @@ def finalize_merge
   def create
     @is_admin = true            # needed for choosing correct method in 'new' tmpl
     @customer = Customer.new(params[:customer])
+    @customer.password = params[:password] unless params[:password].blank?
```

Comment:
Not clear about the logic here. 
1.Are you trying to implement that customer password cannot be created unless there password and password confirmation are both with value? It should be like unless ( term1 and term2)
2. I think they don't need a password if they are creating by a stuff. SO this logic is unecessary.

Reply:
You might not need the params[:password].blank? part. I'll take it out and see if it causes any errors in the tests